### DISCUSSION TITLE
API to get "exportable" version of the dashboards

### DISCUSSION THREAD

**Discussion by roidelapluie** [grafana/grafana#44207] [Dashboards, Panels & Visualizations]
Currently, the dashboard API returns the "non-exportable" version of the dashboard, e.g. it does not include the __inputs section.

Since grafana 8.3, it seems that the dashboards do not return datasources with their names, but with their UID's. Therefore, a dashboard exporter with the API can not be imported in another grafana instance like it was in the past.

I would be great to add a "exported" parameter to the dashboard API, ensuring that dashboards can be moved from one grafana server to the other seamlessly.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
